subprojects/docs/src/snippets/dependencyManagement/managingTransitiveDependencies-dependencyAlignmentWithPlatform/kotlin/buildSrc/src/main/kotlin/myproject.java-library-conventions.gradle.kts

```
@@ -0,0 +1,12 @@
+plugins {
+    id("java-library")
+    id("myproject.publishing-conventions")
+}
+
+publishing {
+    publications {
+        create("maven", MavenPublication::class.java) {
```

We can make this more Kotlin DSL idiomatic now:

```suggestion
        create<MavenPublication>("maven") {
```